runtime/tests/js/webapis_tests.js

```diff
@@ -31,9 +63,9 @@ test("URL", () => {
 // A dummy wrapper to create isolated scopes for individual tests
 // We should eventually replace this with a proper test runner
 // See https://github.com/filecoin-station/zinnia/issues/30
-function test(name, fn) {
+async function test(name, fn) {
```

Comment:
❗ 

Now that `test` is async, you should update all existing tests to `await test` too. We don't want dangling promises, do we?

Reply:
I tested this, without `await` a failing test still produces a failure. Do you think we should still add the awaits?

Reply:
Interesting. IIUC, when a dangling promise is rejected, we enter the "unhandled rejection" path and eventually fail the test. I guess that's fine.

But why are you awaiting some tests and don't await other tests?

Is it possible to be more consistent and use the same approach everywhere?